config/cesm/config_files.xml

```diff
@@ -108,9 +108,11 @@
     <type>char</type>
     <values>
       <value component="datm" comp_interface="nuopc" >$SRCROOT/components/cdeps/datm</value>
-      <value component="datm" comp_interface="mct"   >$CIMEROOT/src/components/data_comps_$COMP_INTERFACE/datm</value>
-      <value component="satm"                        >$CIMEROOT/src/components/stub_comps_$COMP_INTERFACE/satm</value>
-      <value component="xatm"                        >$CIMEROOT/src/components/xcpl_comps_$COMP_INTERFACE/xatm</value>
+      <value component="datm" comp_interface="mct"   >$SRCROOT/components/cpl7/components/data_comps_$COMP_INTERFACE/datm</value>
+      <value component="satm" comp_interface="nuopc" >$CIMEROOT/src/components/stub_comps_$COMP_INTERFACE/satm</value>
+      <value component="satm" comp_interface="mct"   >$SRCROOT/components/cpl7/components/stub_comps_$COMP_INTERFACE/satm</value>
+      <value component="xatm" comp_interface="nuopc" >$CIMEROOT/src/components/xcpl_comps_$COMP_INTERFACE/xatm</value>
```

Comment:
we no longer have stub comps for cmeps - I think this line and the other stubs with comp_interface="nuopc" can be removed.

Reply:
done

Reply:
I had to put back in the stub comps for cmeps.  scripts_regression_tests were failing without them.  I believe it's just an issue with matching.